sites/docker-compose.berwyn.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+    hub-be:
+        environment:
+            DRUPAL_APP_URI: https://drupal.digital-hub-stage.hmpps.dsd.io
```

Comment:
DRUPAL_APP_URI only seems to apply to hub-node so shouldn't be needed for the hub-be container.